pyhap/accessory.py

```diff
@@ -162,6 +170,15 @@ def __getstate__(self):
         state["run_sentinel"] = None
         return state
 
+    @staticmethod
+    def _generate_setup_id():
+        chars = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+        rand_bytes = urandom(8)
+        setup_id = ''
+        for x in range(4):
+            setup_id += chars[struct.unpack_from('i', rand_bytes, x)[0] % 26]
```

Comment:
Shouldn't this be %36 (length of chars)?

Reply:
You're right. I was just following the HAP-NodeJS, not thinking.

Reply:
totally fine, no need to reinvent the wheel. And that's the purpose of the review.